contrib/guix/manifest.scm

```diff
@@ -204,38 +205,45 @@ chain for " target " development."))
     (search-our-patches "nsis-gcc-10-memmove.patch"
                         "nsis-disable-installer-reloc.patch")))
 
-(define (fix-ppc64-nx-default lief)
-  (package-with-extra-patches lief
-    (search-our-patches "lief-fix-ppc64-nx-default.patch")))
-
-;; Our python-lief package can be removed once we are using
-;; guix 83bfdb409787cb2737e68b093a319b247b7858e6 or later.
-;; Note we currently use cmake-minimal.
+;; The python-lief package v0.12.3 is available in
+;; guix 83bfdb409787cb2737e68b093a319b247b7858e6.
+;; However, it:
+;;  - requires a patch; see https://github.com/lief-project/LIEF/pull/718
```

Comment:
I think you can remove this line about needing a patch, as that's not a reason to not use a Guix package, because we can just apply any patches ourselves. The cmake line is good.